tests/gcp/installer/util/test_resource/deploy-forseti-client.yaml.in

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
This is a file level suggestion for everything in ../installer/util/test_resource/

The convention is `test_data/a_name_that_describes_what_it_is.extension`

So we could expect a file like `tests/gcp/installer/util/test_data/invalid_hostname_deploy_forseti_client.yaml.in`